lib/modules/newCommentCount.js

```diff
@@ -40,6 +40,19 @@ module.options = {
 		value: true,
 		description: 'Show the Subscribe button?',
 	},
+	monitorPostsVisited: {
+		type: 'boolean',
+		value: true,
+		description: 'Monitor the number of comments on posts you have visited.',
+		advanced: true,
+	},
+	monitorPostsVisitedIncognito: {
+		dependsOn: 'monitorPostsVisited',
+		type: 'boolean',
+		value: false,
```

Comment:
In my mind that warning isn't an excuse for extensions to misbehave (by saving browsing history when users have implicitly requested not to have that data saved), it's just a warning to users that some extensions *might* and Chrome has no way of preventing it.